src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/DlsFlsValveImpl.java

```diff
@@ -253,4 +260,58 @@ public void accept(StringTerms.Bucket bucket) {
             this.bucket = bucket;
         }
     }
+
+    private static class StringTermsGetter {
+        private static final Field REDUCE_ORDER = getField(InternalTerms.class, "reduceOrder");
```

Comment:
Curious why we are using the InternalTerms.class instead of the previous StringTerms.class

Reply:
Nvm, I see that StringTerms is a child class of InternalTerms.

Reply:
For `getDeclaredField()` to find the field it needs to be called for the class where the field is declared. In case of `reduceOrder` it is declared in `InternalTerms.class` and is accessible from subclasses (`StringTerms.class`)